app/views/layouts/mailer.html.erb

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <style>
+      /* Email styles need to be inline */
```

Comment:
get out of the habit of leaving comments in your code...it will help you during internship 🙄  (bc I do the same)